roles/prometheus/tasks/main.yml

```diff
@@ -11,6 +11,20 @@
     state: directory
     mode: "0755"
 
+- name: create alarm rules dir
+  file:
+    path: "{{ prometheus_config_dir }}/rules"
+    state: directory
+    mode: "0755"
+
+- name: install alarm rules file
+  template:
+    src: templates/alert_rules.yml.j2
```

Comment:
Similar to the comment on `roles/alertmanager/tasks/main.yml`, please make this a variable to allow the user to easily substitute in their own rules file. E.g.,

```
- name: install alarm rules file
  template:
    src: "{{ alert_rules_src }}"
```

Reply:
Thanks, I fixed not only this part but also [Prometheus config part](https://github.com/NVIDIA/deepops/pull/1198/commits/2927a595ca91390e0a2568a1473cdd341313b80d).